AstSemantics.md

```diff
@@ -285,13 +284,15 @@ before any others.
 
 ### Yielding values from control constructs
 
-The `nop`, `if`, `br`, `br_if`, and `return` constructs do not yield values.
+The `nop`, `br`, `br_if`, `br_table`, and `return` constructs do not yield values.
 Other control constructs may yield values if their subexpressions yield values:
 
-* `block`: yields either the value of the last expression in the block or the result of an inner `br` that targeted the label of the block
-* `loop`: yields either the value of the last expression in the loop or the result of an inner `br` that targeted the end label of the loop
-* `if_else`: yields either the value of the true expression or the false expression
+* `block`: yields either the value of the last expression in the block or the result of an inner branch that targeted the label of the block
+* `loop`: yields either the value of the last expression in the loop or the result of an inner branch that targeted the end label of the loop
+* `if`: yields either the value of the last *then* expression or the last *else* expression or the result of an inner branch that targeted the label of one of these.
```

Comment:
This isn't really true. Both the `then` and `else` expressions have to yield values for an `if` to yield a value.